packages/amplify-provider-awscloudformation/src/configuration-manager.ts

```diff
@@ -858,7 +858,7 @@ async function determineAuthFlow(context: $TSContext, projectConfig?: ProjectCon
 
   const generalCreds = projectConfig?.configLevel === 'general';
 
-  if (generalCreds) {
+  if (generalCreds || cfnParams?.configLevel === 'general') {
```

Comment:
Instead of modifying this check, I'd prefer if the const above included this logic.

```ts
  const generalCreds = projectConfig?.configLevel === 'general' || cfnParams?.configLevel === 'general';
```